.github/workflows/daily-smoke-test.yml

```diff
@@ -4,25 +4,25 @@ on:
   schedule:
     # Cron syntax is "minute[0-59] hour[0-23] date[1-31] month[1-12] day[0-6]". '*' is 'any value', and multiple values
     # can be specified with comma-separated lists. All times are UTC.
-    # So this expression means "run at 13:30 UTC every day". This time was chosen because it corresponds to
-    # 8:30AM CDT, meaning that any issues will be surfaced towards the start of business.
-    - cron: "30 13 * * *"
+    # So this expression means "run at 17:30 UTC every day". This time was chosen because it corresponds to
+    # 11:30AM CDT, meaning that any issues will be surfaced towards the start of business.
+    - cron: "30 17 * * *"
 
 jobs:
   # Step 1: Build the scanner tarball so it can be installed locally.
   build-scanner-tarball:
     name: 'Build scanner tarball'
     uses: ./.github/workflows/build-scanner-tarball.yml
     with:
-      target-branch: 'dev'
+      target-branch: 'dev-4'
```

Comment:
I think the validate-pr yml also needs updating.

Reply:
https://github.com/forcedotcom/sfdx-code-analyzer-vscode/blob/dev/.github/workflows/validate-pr.yml#L24